### PR TITLE
Update 'Attend KubeCon' buttons

### LINKED ISSUE #28122: 'Attend KubeCon NA' button needs update
**This is a Bug Report**

**Problem:**
In https://kubernetes.io ,
when user clicks `Attend KubeCon NA virtually on November 17-20, 2020`,
image

user lands on [this page](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20),
which is for KubeCon & CloudNativeCon North America 2021.

**Proposed Solution:**
In [en/_index.html](https://github.com/kubernetes/website/blob/master/content/en/_index.html),

[As-is]
```HTML
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20" button id="desktopKCButton">Attend KubeCon NA virtually on November 17-20, 2020</a>
        <br>
        <br>
        <br>
        <br>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu21" button id="desktopKCButton">Attend KubeCon EU virtually on May 4 – 7, 2021</a>
```

[To-be]
```HTML
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Attend KubeCon in Los Angeles on October 12-15, 2021</a>
        <br>
        <br>
        <br>
        <br>
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu21" button id="desktopKCButton">Revisit KubeCon EU 2021</a>
```

**Page to Update:**
https://kubernetes.io

/language en

### PR DESCRIPTION
**Pull request by jihoon-seo**
Resolves #28122

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-28183--kubernetes-io-master-staging.netlify.app

File: content/en/_index.html [hunk 1]
```diff
@@ -43,12 +43,12 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20" button id="desktopKCButton">Attend KubeCon NA virtually on November 17-20, 2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Attend KubeCon in Los Angeles on October 12-15, 2021</a>
```
Comment (kbhawkey) [inline content/en/_index.html:46]: Hi @jihoon-seo .
Should this announcement include "NA"?
Kubecon NA in Los Angeles, CA?

/cc @celestehorgan

Comment (celestehorgan) [reply content/en/_index.html:46]: @kbhawkey 

Answer from marketing: "KubeCon + CloudNativeCon North America" is the full name that should be used :) 

If you can't fit all ofthat on the button (which... I suspect you may be unable to), "KubeCon North America" will do.

**Comment by jihoon-seo** [comment]
I updated as following:

```diff
-Attend KubeCon in Los Angeles on October 12-15, 2021
+Attend KubeCon NA in Los Angeles, CA on October 11-15, 2021
```

@kbhawkey PTAL!

File: content/en/_index.html (hunk 1 above)
Comment (kbhawkey) [reply content/en/_index.html]: Thanks for the information @celestehorgan !

@jihoon-seo , Can you update the announcement with the suggested text?
I think "KubeCon North America" would fit?
Is it essential to include the location?

**Comment by jihoon-seo** [comment]
I updated as following:

```diff
-Attend KubeCon in Los Angeles on October 12-15, 2021
+Attend KubeCon North America on October 11-15, 2021
```

@kbhawkey PTAL!

**Comment by kbhawkey** [comment]
Thank you @jihoon-seo !
/lgtm

**Pull request merged**